021-IdentityForApps/Student/04-l14n.md

```diff
@@ -10,15 +10,16 @@ CMC IT Leadership is _extremely_ impressed with your SignUp / SignIn flow and ho
 
 First off, one of your developers has developed a _harness application_ where you can incorporate your SignUp / SignIn User Flow. Your developer has parameterized it so that you should only have to modify your application's settings in order to incorporate the User Flow.
 
-**NOTE:** This web application is located in the folder named **HarnessApp** in your Teams channel Files tab. 
+**NOTE:** This web application is located in the **HarnessApp** folder under the [student resources](Resources/HarnessApp) or in your Teams channel Files tab.
 
 ### Custom Templates and Themes
 
 CMC IT Leadership has decided that, while Slate Gray is a nice theme and all, it's pretty plain. They have asked you to add some custom branding, such as corporate logo and background images, and also to use a different layout for the SignUp / SignIn flow.
 
 CMC IT Leadership would like you to use the provided _HTML layout templates, styles, and images_. You will not need to modify the template unless you want, but you will have to find a place to host these resources. Also, examine the resources in this folder and understand how the templates, CSS files, and images are related. There's a marker for a storage account name in several files, so you may need to replace that at some point.
 
-**NOTE:** These files are located in the folder named **PageTemplates** in your Teams channel Files tab.
+**NOTE:** These files are located in the **PageTemplates** folder under the [student resources](Resources/PageTemplates) or in your Teams channel Files tab.
```

Comment:
As per the comment above, please remove the link to the WTH repo.

Reply:
Fixed!